src/main/java/games/strategy/triplea/ui/UnitChooser.java

```diff
@@ -328,7 +322,6 @@ public void changedValue(final ScrollableTextField field) {
   private static final class ChooserEntry {
     private final UnitCategory category;
     private final ScrollableTextFieldListener hitTextFieldListener;
-    private final GameData gameData;
```

Comment:
This field was no longer used after removing the `GameData` parameter from the constructor.